torch/nn/modules/conv.py

```diff
@@ -42,9 +43,9 @@ def reset_parameters(self):
         for k in self.kernel_size:
             n *= k
         stdv = 1. / math.sqrt(n)
-        self.weight.data.uniform_(-stdv, stdv)
+        init.uniform_(self.weight, -stdv, stdv)
```

Comment:
nit: ideally we would just use the corresponding initialization method here (I didn't check, but it's something like glorot or something), so that we can get rid of the stdv etc. This would also simplify the explanations

Reply:
Unfortunately, this is not a special technique. Glorot's init uses `fan_in` and `fan_out` and so does He's init. I don't think this is a named initialization scheme.

Reply:
I believe this one is equivalent to
```python
kaiming_uniform_(self.weight, a=math.sqrt(5))
```